gateway/proxy_eth.go

```diff
@@ -142,6 +142,9 @@ func (gw *Node) checkBlkParam(ctx context.Context, blkParam string, lookback eth
 			break
 		}
 		num = ethtypes.EthUint64(head.Height()) - lookback
+	case "safe":
+		safeEpochDelay := 30 // https://github.com/filecoin-project/FIPs/blob/master/FRCs/frc-0089.md
```

Comment:
How about we make this a const in chain/types/ethtypes/eth_types.go, it's already used by both files

And we should be clear about it in the comment, other than just pointing to the FRC, just because we're not documenting these properly anywhere. Something like:

> Research into Filecoin chain behaviour suggests that probabilistic finality generally approaches the intended stability guarantee at, or near, 30 epochs. Although a strictly "finalized" safe recommendation remains 900 epochs.
> See https://github.com/filecoin-project/FIPs/blob/master/FRCs/frc-0089.md